internal/tiltfile/docker_compose.go

```diff
@@ -293,6 +282,56 @@ func parseDCConfig(ctx context.Context, dcc dockercompose.DockerComposeClient, c
 	return services, nil
 }
 
+func getConfigAndServiceNames(ctx context.Context, dcc dockercompose.DockerComposeClient,
+	configPath string) (conf dcConfig, svcNames []string, err error) {
+	// calls to `docker-compose config` take a bit, and we need two,
+	// so do them in parallel to make things faster
+	configGot := make(chan error, 1)
```

Comment:
Could we make these unbuffered channels? I know it's guaranteed that each channel will only receive one thing but I find the behavior of unbuffered channels easier to reason about.